### PR TITLE
Update screenshot tests

### PR DESCRIPTION
**Pull request by npark-mozilla**
### Pull Request checklist

- [x] This PR includes thorough **tests** or an explanation of why it does not
- [x] This PR includes a **CHANGELOG entry** or does not need one
- [x] I have considered adding **QA labels** on the associated issue (not this PR; `qa-ready` or `qa-denied`)

### CODE REVIEW

File: app/src/androidTest/java/org/mozilla/tv/firefox/ui/screenshots/TooltipCaptureTest.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+        // forward button
+        onView(withId(R.id.tooltip)).inRoot(isPlatformPopup()).check(matches(isDisplayed()));
+        takeScreenshotsAfterWait("tooltip-forwardbutton", 500);
+        device.pressDPadRight();
+        onView(withId(R.id.tooltip)).inRoot(isPlatformPopup()).check(matches(isDisplayed()));
+        takeScreenshotsAfterWait("tooltip-reload", 500);
+        device.pressDPadRight();
+        onView(withId(R.id.tooltip)).inRoot(isPlatformPopup()).check(matches(isDisplayed()));
+        takeScreenshotsAfterWait("tooltip-pintohs", 500);
```
Comment (liuche) [inline app/src/androidTest/java/org/mozilla/tv/firefox/ui/screenshots/TooltipCaptureTest.java]: nit: is this a typo for filename?

File: app/src/androidTest/java/org/mozilla/tv/firefox/ui/screenshots/SettingsTest.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+
+
+public class SettingsTest extends ScreenshotTest {
+
+    private UiDevice mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation());
+
+    @ClassRule
```
Comment (liuche) [inline app/src/androidTest/java/org/mozilla/tv/firefox/ui/screenshots/SettingsTest.java]: Oh! did this not work with a kotlin file?

File: app/src/androidTest/java/org/mozilla/tv/firefox/ui/screenshots/TooltipCaptureTest.java (hunk 1 above)
Comment (npark-mozilla) [reply app/src/androidTest/java/org/mozilla/tv/firefox/ui/screenshots/TooltipCaptureTest.java]: ah actually that's a shorthand for "pinto**H**ome**S**creen"

File: app/src/androidTest/java/org/mozilla/tv/firefox/ui/screenshots/SettingsTest.java (hunk 2 above)
Comment (npark-mozilla) [reply app/src/androidTest/java/org/mozilla/tv/firefox/ui/screenshots/SettingsTest.java]: No it did not - it exhibited two issues,
- It did not run on anything other than EN-US locale.  (i.e. locale could not be switched via fastlane)
- It took duplicate screenshots (two of every kind)

Comment (liuche) [reply app/src/androidTest/java/org/mozilla/tv/firefox/ui/screenshots/SettingsTest.java]: Ah, I guess I originally patterned this off of PocketRecommendationTest but that's ignored, so maybe that test didn't work either! Optionally could add a comment for that test (which is also a kotlin test)

Comment (npark-mozilla) [reply app/src/androidTest/java/org/mozilla/tv/firefox/ui/screenshots/SettingsTest.java]: ah, should have read your comments before I merged... that's right, I should have converted them to java before ignoring them, but that was ideal world scenario :)

**Pull request merged**